icu4j/main/classes/core/src/com/ibm/icu/text/ListFormatter.java

```diff
@@ -98,6 +105,241 @@ public String getName() {
 
     }
 
+    /**
+     * Type of meaning expressed by the list.
+     *
+     * @draft ICU 67
+     * @provisional This API might change or be removed in a future release.
+     */
+    public enum ListFormatterType {
+        /**
+         * Conjunction formatting, e.g. "Alice, Bob, Charlie, and Delta".
+         *
+         * @draft ICU 67
+         * @provisional This API might change or be removed in a future release.
+         */
+        AND,
+
+        /**
+         * Disjunction (or alternative, or simply one of) formatting, e.g.
+         * "Alice, Bob, Charlie, or Delta".
+         *
+         * @draft ICU 67
+         * @provisional This API might change or be removed in a future release.
+         */
+        OR,
+
+        /**
+         * Formatting of a list of values with units, e.g. "5 pounds, 12 ounces".
+         *
+         * @draft ICU 67
+         * @provisional This API might change or be removed in a future release.
+         */
+        UNIT
+    };
+
+    /**
+     * Verbosity level of the list patterns.
+     *
+     * @draft ICU 67
+     * @provisional This API might change or be removed in a future release.
+     */
+    public enum ListFormatterWidth {
+        /**
+         * Use list formatting with full words (no abbreviations) when possible.
+         *
+         * @draft ICU 67
+         * @provisional This API might change or be removed in a future release.
+         */
+        WIDE,
+
+        /**
+         * Use list formatting of typical length.
+         *
+         * @draft ICU 67
+         * @provisional This API might change or be removed in a future release.
+         */
+        SHORT,
+
+        /**
+         * Use list formatting of the shortest possible length.
+         *
+         * @draft ICU 67
+         * @provisional This API might change or be removed in a future release.
+         */
+        NARROW,
+    };
+
+    /**
+     * Class for span fields in FormattedDateInterval.
+     *
+     * @draft ICU 64
+     * @provisional This API might change or be removed in a future release.
+     */
+    public static final class SpanField extends UFormat.SpanField {
+        private static final long serialVersionUID = 3563544214705634403L;
+
+        /**
+         * The concrete field used for spans in FormattedList.
+         *
+         * Instances of LIST_SPAN should have an associated value, the index
+         * within the input list that is represented by the span.
+         *
+         * @draft ICU 64
+         * @provisional This API might change or be removed in a future release.
+         */
+        public static final SpanField LIST_SPAN = new SpanField("list-span");
+
+        private SpanField(String name) {
+            super(name);
+        }
+
+        /**
+         * serizalization method resolve instances to the constant
+         * ListFormatter.SpanField values
+         * @draft ICU 64
```

Comment:
Please change this to `@internal`

Reply:
Done, although I seem to remember saying that we would follow precedent and put these serialization methods on track to stable like we do in other cases.  I don't have an opinion either way.